packages/components/src/components/contract-card/contract-card-items/contract-card-sell.jsx

```diff
@@ -14,24 +14,19 @@ const ContractCardSell = ({ contract_info, getCardLabels, is_sell_requested, onC
         ev.preventDefault();
     };
 
+    if (!should_show_sell) return <React.Fragment />;
```

Comment:
why we are returning empty fragment? isn't to return `null` instead?

Reply:
> why we are returning empty fragment? isn't to return `null` instead?

I did it because component is waiting JSX